tictactoe.js

```diff
@@ -27,200 +27,207 @@
-  scoreboard.innerText = "SCORE \n\nPLAYER 1: " + playerOneWins
-                       + "\n\nPLAYER 2: " + playerTwoWins
+  currentPlayerIndicator.innerText = tieGame
+    ? "CATS GAME"
+    : "Player " + (currentPlayer + 1) + (winner ? " WINS!" : "");
+  scoreboard.innerText =
+    "SCORE \n\nPLAYER 1: " + playerOneWins + "\n\nPLAYER 2: " + playerTwoWins;
 }
 
-function catsGame(){
+function catsGame() {
   let emptyBlocks = 9;
-  [].forEach.call(gameBlocks, function(el){
-    if (el.classList.contains("clicked")){
-    emptyBlocks--
+  [].forEach.call(gameBlocks, function(el) {
+    if (el.classList.contains("clicked")) {
+      emptyBlocks--;
       if (emptyBlocks == 0) {
-        tieGame = true
-        filled = true
+        tieGame = true;
+        filled = true;
       } else {
-        filled = false
+        filled = false;
       }
     }
-  })
+  });
 
-  if (filled && !winner)
-    container.classList.add("cats-game")
+  if (filled && !winner) container.classList.add("cats-game");
 }
 
+function showWinner() {
+  testDiag();
+  testRows();
+  testColumns();
 
-function showWinner(){
-  testDiag()
-  testRows()
-  testColumns()
-
-  if (diagonals){
-    console.log("Slashed!\nPlayer " + (currentPlayer + 1) + " wins!!" )
-    winner = true
-    currentPlayer == 1 ? playerTwoWins++ : playerOneWins++
+  if (diagonals) {
+    console.log("Slashed!\nPlayer " + (currentPlayer + 1) + " wins!!");
+    winner = true;
+    currentPlayer == 1 ? playerTwoWins++ : playerOneWins++;
   } else if (rows) {
-    console.log("ROWS!!\nPlayer " + (currentPlayer + 1) + " wins!!" )
-    winner = true
-    currentPlayer == 1 ? playerTwoWins++ : playerOneWins++
+    console.log("ROWS!!\nPlayer " + (currentPlayer + 1) + " wins!!");
+    winner = true;
+    currentPlayer == 1 ? playerTwoWins++ : playerOneWins++;
   } else if (columns) {
-    console.log("Staright uP!\n Player " + (currentPlayer + 1) + " wins!!" )
-    winner = true
-    currentPlayer == 1 ? playerTwoWins++ : playerOneWins++
+    console.log("Staright uP!\n Player " + (currentPlayer + 1) + " wins!!");
+    winner = true;
+    currentPlayer == 1 ? playerTwoWins++ : playerOneWins++;
   } else {
-    console.log("nice move")
+    console.log("nice move");
   }
 
-  if (winner)
-    container.classList.add("winner")
-
-  catsGame()
+  if (winner) container.classList.add("winner");
 
+  catsGame();
 }
 
-function testDiag(){
-  let downRight = gameBlocks[0].innerText != "" && gameBlocks[0].innerText == gameBlocks[4].innerText   //DOWN RIGHT \
-        && gameBlocks[0].innerText == gameBlocks[8].innerText
-  let upLeft = gameBlocks[2].innerText != "" && gameBlocks[2].innerText == gameBlocks[4].innerText     //UP LEFT /
-        && gameBlocks[2].innerText == gameBlocks[6].innerText
-  if (downRight || upLeft){
-    diagonals = true
-    console.log("three in a row diagonally.")
- }else
-   diagonals = false
+function testDiag() {
+  let downRight =
```

Comment:
This can be improved. Try and write it so that it is automated. Hint you might want to query for each row at the top and then you can iterate through the rows.